frontend/src/pages/Graph/GraphToolbar/GraphFind.tsx

```diff
@@ -1001,7 +1001,7 @@ export class GraphFindComponent extends React.Component<GraphFindProps, GraphFin
         };
       case 'sc':
       case 'sidecar':
-        return { target: 'node', selector: isNegation ? `[?${NodeAttr.hasMissingSC}]` : `[^${NodeAttr.hasMissingSC}]` };
+        return { target: 'node', selector: isNegation ? `[?${NodeAttr.isOutOfMesh}]` : `[^${NodeAttr.isOutOfMesh}]` };
```

Comment:
This needs to be updated in the graphhelpfind.tsx as well.  The `sidecar` and `sc` should be replaced with `outofmesh` and `om`, although we can keep `sidecar` and `sc` for back compat (undocumented), just in case someone is using them in a stored  expression.

Reply:
Thanks for pointing at this! I didn't realized about it.